src/orchestrator/streaming/aggregator.rs

```diff
@@ -142,7 +142,9 @@ impl ResultActor {
         result.token_classification_results.output = Some(detections);
         if input_start_index == 0 {
             // Get input_token_count and seed from first generation message
-            let first = generations.first().unwrap();
+            let first = generations.first().expect(
+                "error retrieving first element in classified generated text stream result",
+            );
```

Comment:
I think this is ok for now, but there should always be a first generation by the time we get to this point.

Initial thoughts for a potential future enhancement:

Above, we request generations from the `GenerationActor`:
```rust
// Get subset of generation responses relevant for this chunk
let generations = self
    .generation_actor
    .get_range(input_start_index, input_end_index)
    .await;
```
Currently, `get_range()` can panic if the start or end index is out of range. We should probably return `Option` and retry if `Option::None` or something until we have the generations needed.